xgi/classes/hypergraph.py

```diff
@@ -512,10 +513,20 @@ def add_edges_from(self, ebunch_to_add, **attr):
 
         Parameters
         ----------
-        ebunch_to_add : iterable of edges
-            Each edge given in the iterable will be added to the
-            graph. Each edge must be given as as an iterable of nodes
-            or an iterable with the last entry as a dictionary.
+        ebunch_to_add : iterable
+
+            An iterable of edges, in one of the following formats.
```

Comment:
These formats refer to each edge in the iterable of edges that is the input, not the whole input, right? I would make that a tad more explicit. Maybe
"An iterable of edges, where each edge must be in one of the following formats (basically, an iterable of nodes with extra options):

Reply:
Good catch, I updated the docs.